mvpa2/mappers/glm/statsmodels_glm.py

```diff
@@ -58,5 +58,6 @@ def _fit_model(self, ds, X, reg_names):
                 add_constant=False,
                 model_gen=self.model_gen)
         res = mod(ds)
-        res.sa[self.get_space()] = reg_names
+        if self.result_expr == 'params':
```

Comment:
we have argument 'results' but storing as 'results_expr' attribute.  I wondered if we could unify one way or another.
Could there be a unittest which quickly demonstrated previous problem or too much hassle?